### DISCUSSION TITLE
Allow non existant entity's to be entered in entity settings

### DISCUSSION THREAD

**Discussion by Mikefila** [zachowj/node-red-contrib-home-assistant-websocket#605] [Feature Request]
I know this is a little backward but it just makes it tough to help others. I need to define entities that don't exist on my system so I can export working flows for others. Now it deletes the entity and sets one of mine as the default.

**Comment by zachowj** [comment]
I had already thought about this. I don't want to add it as the default behavior. Still looking for a good option.

The two that I have thought of: 

- Option in the server config to allow all custom strings
- Some sort of key combination <ctrl-enter>

**Comment by Mikefila** [comment]
I'd prefer it in the server config. I'd think anyone who would have a need for this, wouldn't be too concerned about misspelling entities.

**Reply by zachowj** [reply]
The current implementation is not only about spelling but about filtering the correct ids. Some services define what ids they accept based on the component that created them, domain, and device_class. https://www.home-assistant.io/docs/blueprint/selectors/#entity

**Reply by zachowj** [reply]
Maybe prefix the entity id with a character such as `#`

**Reply by zachowj** [reply]
![chrome_7XXPro64FO](https://user-images.githubusercontent.com/37859597/155250022-8e4f71be-5eaf-4490-8707-a46d1db651a2.gif)
